openedx/core/djangoapps/discussions/docs/decisions/0005-in-context-discussion-course-blocks.rst

```diff
@@ -0,0 +1,39 @@
+Making Linked In-Context Discussions Available to MFEs
+======================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+In the ADR 0004 we investigated how to link a discussion to a unit. We
+also need some way to make these linked discussions available to the
+frontend so they can display those in-context discussions in the correct
+view.
+
+
+Requirements
+------------
+
+An API to access linked discussions for a Unit.
+
+
+Decision
+--------
+
+A direct link to the topic that needs to be embedded can be generated
+by edx-platform and provided to MFEs via the course blocks API which is
+already is used by the learning MFE. The learning MFE can then directly
```

Comment:
nit: `which is already is used by` -> `which is already in use by` or `which is already used by`

Reply:
can you share an example how the new url will be embedded?  at which level in the course blocks api?

Reply:
I've added code examples.